SubjuGator/command/subjugator_launch/config/adaptive_controller.yaml

```diff
@@ -1,6 +1,7 @@
 ---
-kp: 1000, 1000, 1000, 5000, 5000, 5000
-kd: 150, 150, 150, 50, 100, 25
-kg: 5,5,5,5,5,5
-ki: 5,5,5,5,10,5
-use_learned: false
+kp: 120, 150, 200, 100, 50, 50
+kd: 25, 25, 25, 25, 25, 25
+#kd: 0, 0, 0, 0, 0, 0
```

Comment:
not needed

Reply:
Just to clarify, is this for the commented out kd values?

Reply:
I went ahead and deleted those values in 876436d anyways.